examples/express/src/server.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+const openapi = require('@wesleytodd/openapi');
+const app = express();
+const port = 8080;
+
+const API_VERSION_HEADER_NAME = 'API-Version';
+const API_VERSION_HEADER_SCHEMA_NAME = 'apiVersionHeader';
+const API_VERSION_HEADER_SCHEMA = {
+  description: 'De huidige versie van de applicatie',
+  style: 'simple',
+  schema: {
+    type: 'string'
+  }
+};
+
+const oapi = openapi({
+  openapi: '3.0.0',
+  info: {
+    title: 'Express example',
+    description: 'Example project gebouwd met Express',
+    version: '0.0.1',
+    contact: {
+      name: 'API Support',
+      url: 'https://example.com/api/support',
+      email: 'apisupport@example.com'
+    }
+  },
+  servers: [
+    {
+      url: 'https://example.com/api/v1'
+    }
+  ],
+  tags: [
```

Comment:
Dit is niet meer nodig als https://github.com/wesleytodd/express-openapi/pull/71 zou mergen